docs/flocker-features/flockerctl.rst

```diff
@@ -31,14 +26,16 @@ Install it with:
 
    curl -sSL https://get.flocker.io |sh
 
-If you use the :ref:`labs-installer` to install your cluster ``flockerctl`` will work with the ``cluster.yml`` file that it generates and figure out how to connect to your cluster based on that.
-
-If you have used an alternative method to create your cluster, such as a manual installation or our :ref:`CloudFormation installer <cloudformation>`, you will need to set some environment variables for ``flockerctl`` to pick up:
+If you used our :ref:`CloudFormation installer <cloudformation>` to create your cluster, or an alternative method such as a manual installation, you will need to set some environment variables for ``flockerctl`` to pick up:
```

Comment:
:question: I'd remove the discussion of installation methods here and start with:
> You will need to set some environment variables describing the address and credentials that ``flockerctl`` should use to connect to the Flocker control service: